include/fw_info.h

```diff
@@ -27,22 +27,36 @@ extern "C" {
 
 #define MAGIC_LEN_WORDS (CONFIG_FW_INFO_MAGIC_LEN / sizeof(u32_t))
 
-struct fw_info_abi;
 
-/**@brief Function that returns an ABI.
- *
- * @param[in]    id      Which ABI to get.
- * @param[in]    index   If there are multiple ABIs available with the same ID,
- *                       retrieve the different ones with this.
- * @param[out]   abi     Pointer to the abi with the given id and index.
- *
- * @retval 0        Success.
- * @retval -ENOENT  id not found.
- * @retval -EBADF   index too large.
- * @retval -EFAULT  abi was NULL.
+
+/**
+ * This struct is used to request an ABI. The bootloader will populate the
```

Comment:
Isn't it possible for any image to provide an ABI? Here it states `The bootloader will populate`

Reply:
The "bootloader" here is any image that boots another image :).
I could change it to "A bootloader will...". Suggestions?

Reply:
This is now described as "a previous boot stage".